src/Screens/HistoryScreen/HistoryScreen.tsx

```diff
@@ -3,18 +3,23 @@ import CustomButton from "components/CustomButton/CustomButton";
 import HeaderPanel from "components/HeaderPanel/HeaderPanel";
 import LayoutWrapper from "components/LayoutWrapper/LayoutWrapper";
 import { useTranslation } from "react-i18next";
+import TodoList from "components/Todo/TodoList/TodoList";
+import { useSelector } from "react-redux";
+import { AppState } from "store/store";
 
 const HistoryToDoScreen = () => {
   const { t } = useTranslation();
   const showMessage = () => {};
   const btnDeleteTasksAll = (
     <CustomButton label={t("delete-task")} onClick={() => showMessage()} />
   );
-
+  const { todos } = useSelector((state: AppState) => state.todosReducer);
+  const finishedTodos = todos.filter((todo) => todo.completed === true);
```

Comment:
should be enough -> const finishedTodos = todos.filter((todo) => todo.completed);